build.sh

```diff
@@ -71,7 +71,7 @@ install_rpms() {
 
     # Add fast-tracked packages here.  We don't want to wait on bodhi for rpm-ostree
     # as we want to enable fast iteration there.
-    yum -y --enablerepo=updates-testing upgrade rpm-ostree
+    yum -y --enablerepo=updates-testing upgrade rpm-ostree bootupd
```

Comment:
honestly it feels like we should be running the bootupd from inside the installed system and not the one from COSA. We had a [similar problem](https://github.com/coreos/coreos-assembler/issues/3156) with grub2 which is why we were using `chroot_run` before for the `grub2-install`

Reply:
Yes, good catch!  This would be a notable regression for RHCOS.  Um... @HuijingHei you and I discussed this I'm pretty sure (but it was way back in February)...not finding it in https://github.com/coreos/bootupd/pull/420 though?

I think you might be right though and our conclusion was that bootupd itself shouldn't be doing a chroot, but should be run in a (privileged) container.  Will look at this.

Reply:
OK, switched to wrap in `chroot_run`! 

Also of note: this is how bootc does it, because in `bootc install` land, the *container installs itself* so you're already running in the target system.

Reply:
> I think you might be right though and our conclusion was that bootupd itself shouldn't be doing a chroot, but should be run in a (privileged) container. Will look at this.

Agree with you.